portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/create-or-modify-organization.jsx.js

```diff
@@ -50,7 +49,9 @@ var CreateOrModifyOrganizationModal = React.createClass({
                                                         organization={this.state.organization}
                                                         step={this.state.step}
                                                         onStepChange={this.onStepChange}
-                                                        fromStore={false} />
+                                                        fromStore={false}
+                                                        org={this.props.org}
```

Comment:
Yes, it's not very well named but it's different data for the same entity. I'll try to find a better way or name.